custom_reports/http_api/definitions/query_request.yaml

```diff
@@ -0,0 +1,123 @@
+    type: object
+    description: |
+      Data filters for the query. Please note that filtering is only available
+      on dimensions at this point. The top-level object must specify AND/OR
+      clause, but any nested objects may either specify another AND/OR
+      clause or a single filter.
+    properties:
+      operator:
+        type: string
+        description: Logical operator (AND/OR) for the clause.
+        enum:
+          - and
+          - or
+      conditions:
+        type: array
+        description: List of filters or recursively nested clauses.
+        items:
+          type: object
+          required:
+            - column_id
+            - condition
+          properties:
+            column_id:
+              type: string
+              description: ID of the column (must be a dimension).
+            transformation_id:
+              type: string
+              description: |
+                Optional ID of the transformation. Must be supported by
+                the column that was specified and must convert the column
+                into another dimension (and not a metric).
+            condition:
+              type: object
+              description: Definition of a the condition for this column.
+              properties:
+                operator:
+                  type: string
+                  description: |
+                    <p>Filtering operator. Available choices depend on
+                    the specified column's type.</p>
+                    <p>String operators: <code>eq</code>, <code>neq</code>,
+                    <code>contains</code>, <code>not_contains</code>,
+                    <code>starts_with</code>, <code>ends_with</code></p>
+                    <p>Numeric operators: <code>eq</code>,
+                    <code>neq</code>, <code>gt</code>, <code>gte</code>,
+                    <code>lt</code>, <code>lte</code>.</p>
+                    <p>Boolean and enum operators: <code>eq</code>,
+                    <code>neq</code>.</p>
+                value:
+                  description: |
+                    Value that should be matched by filter. Value type must
+                    match the type of the specified column.
+              required:
+                - operator
+                - value
+          example: {"column_id":"event_title","condition":{"operator":"contains","value":"test"}}
+    required:
+      - operator
+      - conditions
+  offset:
+    type: integer
+    default: 0
+    minimum: 0
+    description: Number of rows to skip before beginning to return rows.
+  limit:
+    type: integer
+    default: 100
+    minimum: 1
+    maximum: 100000
+    description: Number of rows to return.
+  order_by:
+    type: array
+    description: |
+      Array of sorting directives. Each directive is a 2-element array with
+      0-based colum index and direction). You can sort on more than one column.
+      By default sorts descending by the first metric in query.
```

Comment:
You can add `default` property instead of this sentence.

Reply:
There is no single default property, since it's calculated dynamically depending on how the request is constructed.

Reply:
OK, I thought it was always `default: [[1, "desc"]]`